src/Uno.UI/UI/Xaml/UIElement.Pointers.Android.cs

```diff
@@ -94,11 +94,11 @@ protected sealed override bool OnNativeMotionEvent(MotionEvent nativeEvent, View
 				var args = new PointerRoutedEventArgs(nativeEvent, nativeEvent.ActionIndex, srcElement, this);
 				var argsAction = actionMasked;
 
-				return OnNativeMotionEvent(args, argsAction, isInView);
+				return OnNativeMotionEvent(nativeEvent, args, argsAction, isInView);
 			}
 		}
 
-		private bool OnNativeMotionEvent(PointerRoutedEventArgs args, MotionEventActions action, bool isInView)
+		private bool OnNativeMotionEvent(MotionEvent nativeEvent, PointerRoutedEventArgs args, MotionEventActions action, bool isInView)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unused method parameter 'nativeEvent'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826514)